src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -916,10 +886,10 @@ static int create_sharedmem_resources(const char* dirname, const char* filename,
     return -1;
   }
 
+#if !defined(_AIX)
```

Comment:
Please enable this for AIX too. Seems we just missed "JDK-7007769: VM crashes with SIGBUS writing PerfData if tmp space is full" (I think AIX port was not official then)

Reply:
done